src/Pages/PopularRepositoriesTable/PopularRepositoriesTable.tsx

```diff
@@ -134,44 +139,121 @@ const PopularRepositoriesTable = () => {
     { searchQuery: debouncedSearchValue } as FilterData,
   );
 
+  const repoToRequestItem = (repo: PopularRepository): CreateContentRequestItem => {
```

Comment:
You should take any functions like this (that don't have internal imported values) and plop them in a "helper.ts"